aksel.nav.no/website/pages/eksempler/progress-bar/indeterminate.tsx

```diff
@@ -1,12 +1,23 @@
+import React from "react";
 import { ProgressBar } from "@navikt/ds-react";
 import { withDsExample } from "@/web/examples/withDsExample";
 
 const Example = () => {
+  // const [isIndeterminate, setIsIndeterminate] = React.useState(false);
   return (
-    <div>
-      <p id="indeterminate">Laster opp fil</p>
-      {/* TODO: Remove outcommented duration prop */}
-      <ProgressBar /* duration={6} */ aria-labelledby="indeterminate" />
+    <div style={{ width: "300px" }}>
+      <p id="indeterminate-working">Jobber med saken</p>
+      <ProgressBar
+        // simulated={{
+        //   seconds: 6,
+        //   onTimeout: () => {
+        //     console.log("Ferdig!");
+        //     setIsIndeterminate(true);
+        //   },
+        // }}
+        aria-labelledby="indeterminate-working"
+      />
+      {/* {isIndeterminate && <p>Oi, dette tok lenger tid en forventet!</p>} */}
```

Comment:
Må denne koden fortsatt være kommentert ut?

Reply:
Ja 😞 Fordi `simulated`-APIet ikke finnes ikke i prod før denne PRen er released, men denne filen importerer `ProgressBar` fra prod. `@ts-ignore` og `@ts-expect-error` feiler henholdvis lokalt og i playwright. Så vet ikke hvordan jeg kan få det inn i samme PR egentlig? 🤷‍♂️ Tar i mot forslag.

Reply:
Lokalt må du kjøre `yarn boot` for at det skal virke. I CI skal det bare virke. Prøv, og så kan vi se på eventuelle feilmeldinger sammen.

Reply:
Åja, will try